### PR TITLE
Relocate DSS Package

### PR DESCRIPTION
**Pull request by Daeinar**
Moves the DSS package from `kyber/share/dss` to `kyber/sign/dss`.

### CODE REVIEW

**Comment by ineiti** [comment]
@Daeinar - is this still relevant?

**Comment by Daeinar** [comment]
@ineiti Done

File: stable/script [hunk 1]
```diff
@@ -0,0 +1,8 @@
+SRC="$1"
+DST="$2"
+
+(
+cp -a $SRC/sign/dss $DST/share/
```
Comment (jeffallen) [inline stable/script]: Barf.

Comment (ineiti) [reply stable/script]: All that in the name of backward-compatibility... Will go away in `v3`.

**Pull request merged**